examples/cpp-simple/README.md

```diff
@@ -8,9 +8,24 @@ It will
 - Every 10 seconds, write a log saying "Hi! I'm a Game Server"
 - After 60 seconds, call `SDK::Shutdown()` to shut the server down.
 
+## Running
+
+If you want to run the example code without building the container yourself, you can start a single gameserver by running
+
+```
+kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/master/examples/cpp-simple/gameserver.yaml
```

Comment:
Will this stop the gameserver when it's finished as well?

Reply:
Yes. If you use the fleet example, you will get a constant churning of new game servers as each one exits after ~1m and is replaced by a new one.

Reply:
Ok, so there's no "shutdown" procedure I'd need to do? I don't know much about kubectl, so I'd run this command and leave it running, so as long as that's OK, great :-)

Reply:
I've updated the instructions to explain this, since if you have these questions other folks are likely to as well.